crates/llvm-builder/tests/build.rs

```diff
@@ -134,9 +164,30 @@ fn clone_build_and_clean_emscripten() -> anyhow::Result<()> {
     let test_dir = common::TestDir::with_lockfile(None)?;
     let command = Command::cargo_bin(common::REVIVE_LLVM)?;
     let program = command.get_program().to_string_lossy();
+
+    Command::cargo_bin(common::REVIVE_LLVM)?
+        .current_dir(test_dir.path())
+        .arg("clone")
+        .assert()
+        .success();
+
+    Command::cargo_bin(common::REVIVE_LLVM)?
+        .current_dir(test_dir.path())
+        .arg("build")
+        .arg("--llvm-projects")
+        .arg("clang")
```

Comment:
```suggestion
```

We should only need LLD; this will make the test consume less resources.

Reply:
That's host llvm, you probably meant the next command, right?

Reply:
Yes sorry the host LLVM needs clang and LLVM.